hmda/src/test/scala/hmda/validation/rules/lar/quality/_2020/Q650Spec.scala

```diff
@@ -0,0 +1,19 @@
+package hmda.validation.rules.lar.quality.twentytwenty
+
+import hmda.model.filing.lar.LarGenerators._
+import hmda.model.filing.lar.LoanApplicationRegister
+import hmda.validation.rules.EditCheck
+import hmda.validation.rules.lar.LarEditCheckSpec
+
+class Q650Spec extends LarEditCheckSpec {
+  override def check: EditCheck[LoanApplicationRegister] = Q650
+
+  property("Interest rate should not be greater than 0 but less than 0.5") {
+    forAll(larGen) { lar =>
+      lar.copy(loan = lar.loan.copy(interestRate = "8.5")).mustPass
+      lar.copy(loan = lar.loan.copy(interestRate = "0.2")).mustFail
+      lar.copy(loan = lar.loan.copy(interestRate = "0.4")).mustFail
+      lar.copy(loan = lar.loan.copy(interestRate = "0.5")).mustPass
```

Comment:
We should include a test for "NA" or "Exempt" as well.

Reply:
@kgudel  updated to include more test cases